docs/content/core/volumes.md

```diff
@@ -213,5 +214,115 @@ fin config set DOCKSAL_VOLUMES=none
+docker extension install mutagenio/docker-desktop-extension:latest
+docker context use desktop-linux-mutagen
+fin project reset
+```
+
+  - Then create the cache as instructed here: https://mutagen.io/documentation/docker-desktop-extension
+  - Please note that although you set "DOCKSAL_VOLUMES" to "bind", it is automatically switched when you run the extension.
+  - Also, you can only remove the single cache instance after you have `fin project remove` your project.
+
+#### The Stand Alone Mutagen version
+
+- This option allows for as many sync sessions as you would like and is free. ( https://github.com/mutagen-io/mutagen )
+- Setup and install as follows:
+  - Create a mutagen.yml configuration file in your project directory with contents like:
+
+```yaml
+sync:
+  defaults:
+    flushOnCreate: true
+    ignore:
+      vcs: false
+      paths:
+        - "node_modules"
+        - "*.sql"
+        - "*.sql.gz"
+        - "/private/"
+        - ".history/" # vsCode local history
+        - ".tags"
+        - "artifact_deploy"
+        - ".git"
+        - "mutagen*"
+        - ".gitignore"
+        - "README*"
+        - ".eslint*"
+        - ".editor*"
+        - ".gitattributes"
+        - ".vscode"
+        - "deployment_identifier"
+        - "salt.txt"
+        - ".acquia-cli.yml"
+        - "phpcs.xml"
+        - "/hooks/"
+        - "/patches/"
+        - ".gitlab-ci.yml"
+        - ".DS_Store"
+    permissions:
+      defaultFileMode: 644
+      defaultDirectoryMode: 755
+  myprojectname:
+    alpha: "./"
+    beta: "docker://docker@myprojectname_cli_1/var/www"
+    mode: "two-way-resolved"
+```
+
+- Replace 'myprojectname' (two instances) with the name of your project in the mutagen.yml file.
+- Run the following:
+
+```bash
+brew install mutagen-io/mutagen/mutagen
+fin config set DOCKSAL_VOLUMES=none
+fin config set MUTAGEN=1
+fin project reset
+curl https://raw.githubusercontent.com/nicoschi/mutagen-for-docksal/master/mutagen > .docksal/commands/mutagen
+chmod +x .docksal/commands/mutagen
+fin mutagen start
+fin db import your_db_backup.sql
+```
+
+- The initial synchronization takes several seconds depending on the amount of files in your project.
+- The database import command is necessary since you have now done a "project reset".
+- It is advisable to have the mutagen monitor running for mutagen so you can see what it is doing at any given time as sometimes
+  it will be busy or need to be restarted if it reports errors.
+- Start the Monitor and leave it running (in a seperate terminal window) with
+
+```bash
+mutagen sync monitor -l
+```
+
+As discussed here ( https://github.com/nicoschi/mutagen-for-docksal/#usage ) if you need to restart your project, do it with
+the mutagen helper:
+
+```bash
+fin mutagen restart
+```
+
+- We believe this is the fastest option available for the MacOs.
+- Documentation on the different mutagen commands : https://mutagen.io/documentation/introduction/getting-started#session-management
```

Comment:
Read documentation on the [different mutagen commands](https://mutagen.io/documentation/introduction/getting-started#session-management)